test/distributed/org/apache/cassandra/fuzz/snapshots/SnapshotsTest.java

```diff
@@ -132,21 +132,21 @@ private Pair<SnapshotsHolder, Map<String, String>> generateParams()
                     case 1:
                         if (!state.truncatedSnapshots.isEmpty())
                         {
-                            keyspace = state.rs.pick(state.truncatedSnapshots.keySet()).split("\\.")[0];
+                            keyspace = state.rs.pickUnorderedSet(state.truncatedSnapshots.keySet()).split("\\.")[0];
```

Comment:
in pulling all the `RandomSource` in this method got renamed to be 2 different methods

`pickUnorderedSet` and `pickOrderedSet`... unorded needs the sub type to be `Comparable` and will do a sort so the result is deterministic... `pickOrderedSet` doesn't sort as the set order should be deterministic already... this test uses `HashMap` which is non-determiistic so we need `pickUnordedSet`